packages/conf-gnutls/conf-gnutls.1/opam

```diff
@@ -0,0 +1,7 @@
+opam-version: "1.2"
+maintainer: "vb@luminar.eu.org"
+build: [["pkg-config" "gnutls"]]
+depexts: [
+  [["debian"] ["gnutls-dev"]]
+  [["ubuntu"] ["gnutls-dev"]]
```

Comment:
It [looks](https://travis-ci.org/ocaml/opam-repository/jobs/49844082) like these should say "[`libgnutls-dev`](http://packages.ubuntu.com/precise/libgnutls-dev)".